app/components/Article/FeedbackForm/feedbackForm.css

```diff
@@ -0,0 +1,88 @@
+.feedback-header {
+  color: var(--colors-black);
+}
+.feedback-container {
+  border-radius: 6px;
+  background-color: #fff;
+  box-shadow: 0px 16px 40px rgba(175, 183, 194, 0.2);
+  border: 1px solid #dfe3e9;
+  box-sizing: border-box;
+  width: 384px;
+  height: 542px;
+  padding: var(--spacing-32);
+}
+.select-option {
+  border-radius: 6px;
+  background-color: #fff;
+  box-shadow: 0px 16px 40px rgba(175, 183, 194, 0.2);
+  border: 1px solid #dfe3e9;
+  box-sizing: border-box;
+  width: 320px;
+  height: 48px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  padding: 7px 17px;
+  margin: var(--spacing-16) 0;
+  cursor: pointer;
+}
+.select-option.selected {
+  border: 1px solid var(--colors-teal-500);
+}
+
+.feedback-form {
+  width: 100%;
+  position: relative;
+  height: 542px;
+  text-align: left;
+  font-size: 16px;
+  color: var(--colors-cool-grey-600);
+  font-family: Poppins;
+  z-index: 2;
+}
+
+.feedback-text {
+  width: calc(100% - var(--spacing-24));
+  max-width: calc(100% - var(--spacing-24));
+  height: 128px;
+  border-radius: 6px;
+  box-shadow: 0px 16px 40px rgba(175, 183, 194, 0.2);
+  border: 1px solid var(--colors-cool-grey-200);
+  padding: var(--spacing-12);
+}
+.small {
+  position: relative;
+  letter-spacing: -0.01em;
+  line-height: 170%;
+}
+
+.small3 {
+  position: relative;
+  letter-spacing: -0.01em;
+  line-height: 170%;
+  font-weight: 600;
+}
+.submit-feedback {
+  position: absolute;
+  top: 462px;
+  left: 32px;
+  border-radius: 6px;
+  background-color: var(--colors-teal-500);
+  box-shadow: 0px 16px 40px rgba(32, 44, 89, 0.05);
+  width: 320px;
+  height: 48px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  padding: 7px 17px;
```

Comment:
Always use a variable. If the padding in the file was 7px I meant for it to be 8px sorry! 17px would be 16px.